armada/chartbuilder.py

```diff
@@ -132,7 +132,12 @@ def get_templates(self):
         for root, _, files in os.walk(os.path.join(self.source_directory,
                                                    'templates'), topdown=True):
             for tpl_file in files:
-                templates.append(Template(name=tpl_file,
+                tname = os.path.relpath(os.path.join(root, tpl_file),
+                                        os.path.join(self.source_directory,
+                                                     'templates'))
+                LOG.info("Template name: %s", tname)
```

Comment:
Same, dont need this log or it can be DEBUG with a cleaned up message, e.g. Processing Template: ..